ndk-build/src/manifest.rs

```diff
@@ -29,6 +29,9 @@ pub struct AndroidManifest {
     #[serde(default)]
     pub uses_permission: Vec<Permission>,
 
+    #[serde(default)]
+    pub queries: Vec<Query>,
```

Comment:
Can this element be specified multiple times?

Reply:
I assume not but i can find any definitive claification in the docs, I'll change to only one optional.

Reply:
I thought I had removed this comment again, it seems like this is possible.

Since you have a use case for `<queries>` and have presumably tested this PR, could you perhaps add this element twice (with "bogus" values) and see if `aapt` complains?

Reply:
@korejan Did you manage to test this, seeing as the current revision of this PR uses `Option<Queries>` instead of `Vec<Queries>`?

Reply:
If you mean test the change then yes, I'm using these features for my own project.

Reply:
Have you tested it with _multiple_ `Queries` elements, specifically?

Reply:
No, it changed to an Option type, not the way around and that it is what I'm using.

Reply:
The request is to test that while it is a `Vec`, instead of assuming my question above implies that it does not, nor being confirmed by the lack of definitive documentation on the case.

Reply:
That's a bit harsh isn't it? I'm asking for you to undo and test a simple change, which you seem to have made on my request while I was really just asking because I have no idea either. This should be a trivial thing to test on your end.

> I've never had this experinice with PRs I made to other projects

Maybe their level of quality assurance just isn't as high? I'm also mainly maintaining and contributing to this project in my free time, if I "just leave it" there's unlikely to be someone else with more time on their hands to review and approve your PR. And I don't want to merge in something that requires maintainer cleanup afterwards, nor incomplete/incorrect docs for end users. The only obvious solution is handling it in PR review.